aliases_shared

```diff
@@ -0,0 +1,29 @@
+alias cd.="cd .."
+alias cd2.="cd ../.."
+alias cd3.="cd ../../.."
+alias cd4.="cd ../../../.."
+alias cd5.="cd ../../../../.."
+alias cd6.="cd ../../../../../.."
+
+alias ll='ls -alF'
+alias la='ls -A'
+alias l='ls -CF'
+
+bmux_start () {
```

Comment:
It's true; also a short version without overwriting tmux.

On Monday, October 26, 2015, John DeWyze <notifications@github.com> wrote:

> In aliases_shared
> <https://github.com/dewyze/dotfiles/pull/1#discussion_r43072881>:
>
> > @@ -0,0 +1,29 @@
> > +alias cd.="cd .."
> > +alias cd2.="cd ../.."
> > +alias cd3.="cd ../../.."
> > +alias cd4.="cd ../../../.."
> > +alias cd5.="cd ../../../../.."
> > +alias cd6.="cd ../../../../../.."
> > +
> > +alias ll='ls -alF'
> > +alias la='ls -A'
> > +alias l='ls -CF'
> > +
> > +bmux_start () {
>
> @raymondberg <https://github.com/raymondberg> why bmux? Honorable nod to
> its origin?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dewyze/dotfiles/pull/1/files#r43072881>.
>


-- 
Respectfully,
Ray Berg